src/prelaunch_data_generator/el_cl_genesis/el_cl_genesis_generator.star

```diff
@@ -25,7 +26,15 @@ def generate_el_cl_genesis_data(
     capella_fork_epoch,
     deneb_fork_epoch,
     electra_fork_epoch,
+    latest_block,
 ):
+    files = {}
+    if latest_block == "":
```

Comment:
perhaps the following is cleaner;

```
files = {}
shadowfork_file = ""
if latest_block != "":
    files[SHADOWFORK_FILEPATH] = latest_block
     shadowfork_file = SHADOWFORK_FILEPATH + "/shadowfork/latest_block.json"
```

note lesser indentation; and you don't need the if latest_block condition below either

Reply:
done